code/modules/admin/admin_verbs.dm

```diff
@@ -1148,7 +1237,8 @@
 		if(isobserver(C.mob) || C.mob.stat == DEAD)
 			continue
 		if(ishuman(C.mob))
-			C.mob.loc = get_turf(usr)
+			var/mob/living/carbon/human/H = C.mob
```

Comment:
There is no need to type cast this, as `forceMove()` is defined at the mob level.
This could be improved by replacing the observer check with a `if(!ishuman(C.mob))` and thus avoiding double checks.